amethyst_renderer/src/pipe/effect/pso.rs

```diff
@@ -102,9 +112,10 @@ impl<'d> PipelineInit for Init<'d> {
     }
 }
 
-#[derive(Clone, Debug, Default, Eq, Hash, PartialEq)]
+#[derive(Clone, Debug, Default, PartialEq)]
 pub struct Data {
     pub const_bufs: Vec<<RawConstantBuffer as DataBind<Resources>>::Data>,
```

Comment:
@kvark I only did this to mimic the `gfx_pipeline!` macro included in `gfx`. Personally, I find it clearer that the `Data` struct contains `DataBind::Data` associated types.